users/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class User(AbstractUser):
+    phone = models.CharField(max_length=32, unique=True, blank=True, null=True)
+    email = models.EmailField(max_length=255, unique=True)
+
+    USERNAME_FIELD = "email"
+    REQUIRED_FIELDS = []
+
+
+class Registration(models.Model):
+    email = models.EmailField(max_length=255)
+
+    event = models.ForeignKey(
```

Comment:
Shouldn't this model be in events app as it is closely related to events.

Reply:
I placed it in user's app because the one who is registering is user of our application. I am saving this record in users app that this user is registered in this particular event. The events model only contains details about a particular event.